src/detect-engine-prefilter-common.h

```diff
@@ -94,4 +94,14 @@ PrefilterPacketHeaderExtraMatch(const PrefilterPacketHeaderCtx *ctx,
     return TRUE;
 }
 
+static inline bool PrefilterIsPrefilterableById(const Signature *s, enum DetectKeywordId kid) {
```

Comment:
Please add this to its own commit as well.

Reply:
ok